README.md

```diff
@@ -441,6 +442,41 @@ function* sendFormDataToServer(formData) {
 }
 ```
 
+### You can also add more than default stages with `createExtendedRoutine`:
+```javascript
+import { createExtendedRoutine } from 'redux-saga-routines';
+
+const projects = createExtendedRoutine('projects', 'TOGGLE');
+
+projects.TOGGLE === 'projects/TOGGLE';
+projects.toggle({ id: 42 }) === { type: "projects/TOGGLE", payload: { id: 42 } };
+```
+
+You can pass an array or stages:
+```javascript
+import { createExtendedRoutine } from 'redux-saga-routines';
+
+const other = createExtendedRoutine('other', ['OPEN', 'CLOSE']);
```

Comment:
I'll try to add better docs and examples for `createExtendedRoutine` and `createCustomRoutine`